docs/data/migration/migration-tree-view-v6/migration-tree-view-v6.md

```diff
@@ -253,6 +253,40 @@ you need to use the new `icon` slot on this component:
 Note that the `slots` prop expects a React component, not the JSX element returned when rendering this component.
 :::
 
+### ✅ Use slots to define the group transition
+
+The component used to animate the item children
+is now defined as a slot on the Tree Item component.
```

Comment:
```suggestion
is now defined as a slot on the `TreeItem` component.
```

Reply:
We used "Tree Item" on the icon sections above so I followed the same nomenclature :+1:

Reply:
oh, but you are using "`TreeItem` component" in a section below ... thought this might be the way to go

Reply:
OK so we are already not consistent...
I'll see to unify everything before merging :+1: